scripts/core/editor3/components/Editor3.jsx

```diff
@@ -20,146 +21,137 @@ import Toolbar from './toolbar';
  * @description Editor3 is a draft.js based editor that support customizable
  *  formatting (shortly more features will come ...)
  */
-export class Editor3 extends React.Component {
-    constructor(props) {
-        super(props);
 
-        const initialContentState = stateFromHTML(props.value);
+export const getInitialState = (spellcheckerService, value, onChangeValue, readOnly, showToolbar, editorFormat) => {
```

Comment:
This is confusing. In Redux stores map to props and state is generally not in play. I think having the function here indicates that there is a state to the editor, which is false, this is the state of the store, so I would move it from here to somewhere else.

I would follow the examples provided in the Redux framework.